src/factions/AdeptaSororitas/models.jsonnet

```
@@ -0,0 +1,361 @@
+              "Cost": 13,
+              "NameEn": "Arco-Flagellant",
+              "NameFr": "Arco-Flagellant",
+              "NameDe": "Arco-Flagellant",
+              "IsCommander": false,
+              "NumberOfKnownPsychics": 0,
+              "NumberOfPsychicsManifestationPerRound": 0,
+              "NumberOfPsychicsDenialPerRound": 0,
+              "WarGearOptions": [],
+              "Specialists": utils.Specialists(["Z"]),
+              "ModelProfileWeapons": [],
+              "CostOverrides": [],
+              "LevelCosts": []
+            },
+            {
+                "Id": "END",
+                "Movement": 7,
+                "WeaponSkill": 4,
+                "BallisticSkill": 0,
+                "Strength": 4,
+                "Toughness": 3,
+                "Wounds": 2,
+                "Attacks": 3,
+                "Leadership": 7,
+                "Save": 7,
+                "MaximumNumber": 1,
+                "Cost": 14,
+                "NameEn": "Endurant",
+                "NameFr": "Endurant",
+                "NameDe": "Endurant",
+                "IsCommander": false,
+                "NumberOfKnownPsychics": 0,
+                "NumberOfPsychicsManifestationPerRound": 0,
+                "NumberOfPsychicsDenialPerRound": 0,
+                "WarGearOptions": [],
+                "Specialists": utils.Specialists(["Z"]),
+                "ModelProfileWeapons": [],
+                "CostOverrides": [],
+                "LevelCosts": []
+            }
+          ],
+          "ModelWeapons": [
+            {
+              "WeaponId": "ARCOFL"
+            }
+          ],
+          "WarGearOptions": []
+    },
+    {
+        "Id": "REPS",
+        "NameEn": "Repentia Superior",
+        "NameFe": "Repentia Superior",
+        "NameDe": "Repentia Superior",
+        "KeywordsEn": "IMPERIUM, COMMANDER, INFANTRY, REPENTIA SUPERIOR",
+        "KeywordsFr": "IMPERIUM, COMMANDER, INFANTRY, REPENTIA SUPERIOR",
+        "KeywordsDe": "IMPERIUM, COMMANDER, INFANTRY, REPENTIA SUPERIOR",
+        "ModelProfiles": [
+            {
+              "Id": "REPS",
+              "Movement": 6,
+              "WeaponSkill": 3,
+              "BallisticSkill": 3,
+              "Strength": 3,
+              "Toughness": 3,
+              "Wounds": 4,
+              "Attacks": 4,
+              "Leadership": 8,
+              "Save": 3,
+              "MaximumNumber": 1,
+              "Cost": 30,
+              "NameEn": "Repentia Superior",
+              "NameFr": "Repentia Superior",
+              "NameDe": "Repentia Superior",
+              "IsCommander": true,
+              "NumberOfKnownPsychics": 0,
+              "NumberOfPsychicsManifestationPerRound": 0,
+              "NumberOfPsychicsDenialPerRound": 0,
+              "WarGearOptions": [],
+              "Specialists": utils.Specialists(["FR", "FE", "ME", "F"]),
+              "ModelProfileWeapons": utils.Ids("WeaponId", ["NWHP", "EPT", "GF", "GK"]),
```

```suggestion
              "ModelProfileWeapons": utils.Ids("Weapon", ["NWHP", "EPT", "GF", "GK"]),
```

The utils class appends Id already anyways

---

👍